pages/FAQ.md

```diff
@@ -0,0 +1,38 @@
+# FAQ
+
```

Comment:
Before we dive into the questions, maybe do a "Where do I download slimefun or addons" one where you link to the "Installing-Slimefun" and "Addons" page.